code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -71,6 +71,9 @@
 		'sound/voice/lizard/lizard_scream_3.ogg',
 	)
 
+/datum/species/lizard/get_laugh_sound(mob/living/carbon/human)
+	return 'sound/voice/lizard/lizard_laugh1.ogg'
```

Comment:
```suggestion
	if(!istype(user))
		return
	return 'sound/voice/lizard/lizard_laugh1.ogg'
```

Reply:
That's fine, since I assume you call dna.species.get_laugh_sound()
But it's still safe to check that you are actually passing ``user`` in the args of the proc.
If there is no user being passed, it should not pass a sound

Reply:
I don't think it really matters if `user` is unused by the lizard variant to be honest.